src/components/index.jsx

```diff
@@ -100,7 +126,7 @@ function Home() {
 														<JobLocation>
 															<li>
 																<Badge pill>
-																	<TimeIcon />	{job.createdAt}
+																	<TimeIcon />	{job.createdAt?.toLocaleString()}
```

Comment:
Why do you need the `?` here? Isn't it always there or no?

Reply:
without it its complaining when date is null